lib/Slack_web_api.js

```diff
@@ -1,314 +1,180 @@
 var request = require('request');
 
-module.exports = function(bot, config) {
+function noop() {}
```

Comment:
Yeah, `noop` is short for "no operation", and has become a convention for methods that don't do anything (https://en.m.wikipedia.org/wiki/NOP)